### PR TITLE
[FVM] update EVM integration test to use FVM

### PR DESCRIPTION
**Pull request by ramtinms**

### CODE REVIEW

File: fvm/evm/evm_test.go [hunk 1]
```diff
@@ -49,66 +40,72 @@ func TestEVMRun(t *testing.T) {
...
+						executionSnapshot, output, err := vm.Run(
+							ctx,
+							script,
+							snapshot)
+						require.NoError(t, err)
+						require.NoError(t, output.Err)
+						assert.Equal(t, cadence.Bool(true), output.Value)
+
+						_ = executionSnapshot
+						// snapshot = snapshot.Append(executionSnapshot)
```
Comment (turbolent) [inline fvm/evm/evm_test.go:78]: Do these lines still need to be turned into assertions about the execution state changes?

File: fvm/bootstrap.go [hunk 2]
```diff
@@ -781,6 +781,8 @@ func (b *bootstrapExecutor) setStakingAllowlist(
 }
 
 func (b *bootstrapExecutor) setupEVM(service flow.Address) {
+
+	b.createAccount(nil) // account for storage
```
Comment (turbolent) [inline fvm/bootstrap.go:785]: Is the account later on referred to by address? Is this order dependent?

File: fvm/bootstrap.go [hunk 3]
```diff
@@ -791,6 +793,9 @@ func (b *bootstrapExecutor) setupEVM(service flow.Address) {
 		Transaction(tx, 0),
 	)
 	panicOnMetaInvokeErrf("failed to deploy EVM contract: %s", txError, err)
+	// TODO: clean up
+	// b.accounts.SetContract(stdlib.ContractName, service, stdlib.ContractCode)
+	// TODO: think about storage
```
Comment (turbolent) [inline fvm/bootstrap.go:798]: Can these lines be removed?

File: fvm/bootstrap.go (hunk 2 above)
Comment (ramtinms) [reply fvm/bootstrap.go:785]: this is the address we use for storage of data. (we have the index, so we don't need to capture the address) It is order-dependent but other initial addresses are also like this one, I might refactor this soon.

**Review by turbolent — APPROVED** [review]

**Pull request merged**